EndlessClient/Controllers/FunctionKeyController.cs

```diff
@@ -11,6 +11,7 @@
 using EndlessClient.Rendering.Map;
```

Comment:
Looks like nothing actually changed here, please reset to the content of the master branch